src/Controller/Controller.php

```diff
@@ -206,7 +212,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var string
      */
-    public $plugin;
+    protected $plugin;
```

Comment:
We could but I've seen some tests that set the plugin property. I'm not sure how this would work with immutable requests but this could be a matter of refactoring tests.